Artesian/Artesian.SDK/Service/Clients/Client.cs

```diff
@@ -68,8 +71,8 @@ public Client(IArtesianServiceConfig config, string Url, ArtesianPolicyConfig po
             cfg.ObjectCreationHandling = ObjectCreationHandling.Replace;
 
             _jsonSerializer = new JsonContentSerializer(cfg);
-            _msgPackSerializer = new MessagePackContentSerializer(CustomCompositeResolver.Instance);
-            _lz4msgPackSerializer = new LZ4MessagePackContentSerializer(CustomCompositeResolver.Instance);
+            _msgPackSerializer = new MessagePackContentSerializer(CustomCompositeResolver.Instance ?? throw new InvalidOperationException("CustomCompositeResolver.Instance is null"));
```

Comment:
change CustomCompositeResolver.Instance to be not nullable  and remove exceptions